### PR TITLE
fix: don't cause a wrong redirect when performing community admin

### LINKED ISSUE #11146: Community -> Manage permissions: deleting permission redirects to Overview page
### Description

1. create a permission for community
2. delete it
3. note that user is redirected to Overview page

**Expected**: stay on Permissions page after editing / deleting permissions

https://github.com/status-im/status-desktop/assets/82375995/501861dd-d949-4314-b554-23c48b8e88d6

### PR DESCRIPTION
**Pull request by 0x-r4bbit**
actions

Fixes #11146

### CODE REVIEW

**Review by caybro — COMMENTED** [review]
The code is identical, how does this fix the bug? :thinking:

**Comment by 0x-r4bbit** [comment]
> The code is identical, how does this fix the bug? 

So it seems that moving the actual expression out of the model prevents it from registering a change on the `settingsMenuModel`, which is what we need, because otherwise it'll reset the `currentIndex` of the view which causes the redirect bug.

**Review by caybro — APPROVED** [review]

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 1]
```diff
@@ -51,8 +51,8 @@ StatusSectionLayout {
     property bool hasAddedContacts: false
     property var transactionStore: TransactionStore {}
 
-    property bool isAdmin: community.memberRole === Constants.memberRole.owner ||
-                           community.memberRole === Constants.memberRole.admin
+    property bool isOwner: community.memberRole === Constants.memberRole.owner
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:54]: readonly?

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 2]
```diff
@@ -51,8 +51,8 @@ StatusSectionLayout {
...
+    property bool isAdmin: isOwner || community.memberRole === Constants.memberRole.admin
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:55]: readonly?

**Review by noeliaSD — APPROVED** [review]
LGTM! Just a small comment added!

Tested creating now permission and it works fine! Not tested when minting.

**Pull request merged**